spyder/config/main.py

```diff
@@ -1,729 +1,523 @@
-              'solarized/dark/builtin':    ('#6c71c4', False, False),
-              'solarized/dark/definition': ('#268bd2', True, False),
-              'solarized/dark/comment':    ('#586e75', False, True),
-              'solarized/dark/string':     ('#2aa198', False, False),
-              'solarized/dark/number':     ('#cb4b16', False, False),
-              'solarized/dark/instance':   ('#b58900', False, True)
-             }),
-            ('lsp-server',
-             {
-              # This option is not used with the LSP server config
-              # It is used to disable hover hints in the editor
-              'enable_hover_hints': True,
-              'code_completion': True,
-              'jedi_definition': True,
-              'jedi_definition/follow_imports': True,
-              'jedi_signature_help': True,
-              'preload_modules': PRELOAD_MDOULES,
-              'pyflakes': True,
-              'mccabe': False,
-              'pycodestyle': False,
-              'pycodestyle/filename': '',
-              'pycodestyle/exclude': '',
-              'pycodestyle/select': '',
-              'pycodestyle/ignore': '',
-              'pycodestyle/max_line_length': 79,
-              'pydocstyle': False,
-              'pydocstyle/convention': 'numpy',
-              'pydocstyle/select': '',
-              'pydocstyle/ignore': '',
-              'pydocstyle/match': '(?!test_).*\\.py',
-              'pydocstyle/match_dir': '[^\\.].*',
-              'advanced/command_launch': 'pyls',
-              'advanced/host': '127.0.0.1',
-              'advanced/port': 2087,
-              'advanced/external': False,
-              'advanced/stdio': False
-             })
-            ]
-
-
-# =============================================================================
-# Config instance
-# =============================================================================
-# IMPORTANT NOTES:
-# 1. If you want to *change* the default value of a current option, you need to
-#    do a MINOR update in config version, e.g. from 3.0.0 to 3.1.0
-# 2. If you want to *remove* options that are no longer needed in our codebase,
-#    or if you want to *rename* options, then you need to do a MAJOR update in
-#    version, e.g. from 3.0.0 to 4.0.0
-# 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '50.1.0'
-
-
-# Main configuration instance
-try:
-    CONF = UserConfig('spyder', defaults=DEFAULTS, load=True,
-                      version=CONF_VERSION, subfolder=SUBFOLDER, backup=True,
-                      raw_mode=True)
-except Exception:
-    CONF = UserConfig('spyder', defaults=DEFAULTS, load=False,
-                      version=CONF_VERSION, subfolder=SUBFOLDER, backup=True,
-                      raw_mode=True)
-
-# Removing old .spyder.ini location:
-old_location = osp.join(get_home_dir(), '.spyder.ini')
-if osp.isfile(old_location):
-    os.remove(old_location)
+# -*- coding: utf-8 -*-
```

Comment:
@steff456, please revert the changes made by the original author to the line endings of this file, so we can get a good diff about what really changed in this PR (the current line endings are LF, but the previous ones were CRLF).